### PR TITLE
Feature/various fixes

### PR DESCRIPTION
**Pull request by petersteneteg**

### CODE REVIEW

File: modules/python3/CMakeLists.txt [hunk 1]
```diff
@@ -101,23 +101,54 @@ endif()
...
 # Add Python to package
-if(IVW_PACKAGE_PROJECT AND BUILD_SHARED_LIBS)
+option(IVW_PACKAGE_EMBED_PYTHON "Embed a python dist in the installer.\
+ Otherwise a existing python installation is needed to run inviwo" ON)
+if(IVW_PACKAGE_PROJECT AND BUILD_SHARED_LIBS AND IVW_PACKAGE_EMBED_PYTHON)
     if(WIN32)
+        if(IVW_CMAKE_DEBUG)
+            set(show SHOW_PROGRESS)
+        else()
+            set(shiw "")
```
Comment (r-englund) [inline modules/python3/CMakeLists.txt]: shiw, do you mean show?

Comment (petersteneteg) [reply modules/python3/CMakeLists.txt]: yes...

File: docs/CMakeLists.txt [hunk 2]
```diff
@@ -379,208 +381,188 @@ function(ivw_private_make_help)
...
-    set(dependency_list "")
-
-    set(GENERATE_IMG "YES")
-
-    if(PYTHONINTERP_FOUND)
-        if(${MSVC})
-            set(PREFIX_PYTHON "python ")
-        else()
-            set(PREFIX_PYTHON "python3 ")
+    #string(REPLACE " " "\ " SPACE_PYTHON ${PYTHON_EXECUTABLE})
```
Comment (ResearchDaniel) [inline docs/CMakeLists.txt:416]: Remove

**Review by ResearchDaniel — APPROVED** [review]
Looks good from what I can see

File: docs/CMakeLists.txt (hunk 2 above)
Comment (petersteneteg) [reply docs/CMakeLists.txt:416]: This is some old stuff, that I did not comment out, and it seems a bit shady, so I left it until I have time to investigate it...

**Pull request merged**